esx_service/Makefile

```diff
@@ -33,19 +33,21 @@ OFFLINE_DEPOT := vmware-esx-vmdkops-$(PKG_VERSION).zip
 VIB_BIN  = $(BIN)/$(VIB)
 
 # Shared lib. Note that it is referred by name from vmci_srv.py
-VMCI_SRV_LIB := $(BIN)/libvmci_srv.so
+VMCI_SRV_LIB   := $(BIN)/libvmci_srv.so
+VMCI_SRV_LIB64 := $(BIN)/libvmci_srv64.so
```

Comment:
should it be  something like ? 
```
VMCI_SRV_LIB_NAME   := libvmci_srv.so
VMCI_SRV_LIB32   := $(BIN)/$(VMCI_SRV_LIB_NAME)
VMCI_SRV_LIB64 := $(BIN)/$(VMCI_SRV_LIB_NAME).64
```

since you will be copying it to staging area explicitly to $(VMCI_SRV_LIB_ NAME) anyways ?